src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let amount = days * PRICE_PER_DAY;
```

Comment:
Consider using `const` for the `amount` variable since it's not being reassigned, but merely updated based on the condition. This helps to prevent unintentional reassignment and keeps the code more predictable.